components/x-privacy-manager/readme.md

```diff
@@ -0,0 +1,46 @@
+# x-privacy-manager
+
+This module creates an interface giving users the ability to give or withhold consent to the sale of their data under the provisions of the CCPA (California Consumer Protection Act), as a first step towards the FT's journey towards a Unified Privacy solution.
+
+It is intended for use with Page Kit on FT.com, as a component within FT App, and as a brandable page linked to by Specialist Titles.
```

Comment:
> as a brandable page linked to by Specialist Titles.

How can it be served as a page?

Reply:
I'll amend the wording, but the intention is that it can be consumed by sites outside of ft.com:

> "It is intended for use with Page Kit on FT.com, as a component within FT App, and ~as a~ on standalone pages (e.g. as linked to by Specialist Titles)"